lib/active_merchant/billing/gateways/paybox_direct.rb

```diff
@@ -64,10 +64,31 @@ def initialize(options = {})
         super
       end
 
+      def add_3dsecure(post, options)
+        # ECI=02 => MasterCard success
+        # ECI=05 => Visa, Amex or JCB success
+        if options[:eci] == '02' || options[:eci] == '05'
+          post[:"3DSTATUS"] = 'Y'
+          post[:"3DENROLLED"] = 'Y'
+          post[:"3DSIGNVAL"] = 'Y'
+          post[:"3DERROR"] = '0'
+        else
+          post[:"3DSTATUS"] = 'N'
```

Comment:
Hello @jcpaybox 
I understand other possible combinations of ECI and enrolment statuses, as in the `Test Case 4: Attempts/Non-Participating` Cardinal 1.x test case  [here](https://cardinaldocs.atlassian.net/wiki/spaces/CCen/pages/400654355/3DS+1.0+Test+Cases#Test-Case-4:-Attempts/Non-Participating) are not handled. 
```
Enrolled = Y
EciFlag = 01 or 06
```
Is it intentional?

How is 2.x supported?

Thank you!